### ISSUE TITLE
If responsive mode is `height_ar`, toolbar box `above` or `below` doesn't work

### ISSUE THREAD

**Issue by birdsarah** [bokeh/bokeh#4393]
To reproduce:

``` python
    gridplot([[plot_3]], responsive='height_ar', toolbar_location='above')
```

You'll see the toolbar sort of splatted uglily down the left. I believe it's because the toolbar hasn't got the width it needs.

**Comment by bryevdv** [comment]
Things are probably better after #8085. That's just a guess, but I will close this anyway since there's no complete test case and the `responsive` argument is gone. Please open a new issue with a test case if there is still a demonstrable issue with > 1.1
